cfme/storage/volume.py

```diff
@@ -243,6 +260,22 @@ def create_snapshot(self, name, cancel=False, reset=False):
                 view.save.click()
                 return snapshot_collection.instantiate(name, self.provider)
 
+
+    def attach_instance(self, name, mountpoint=None):
```

Comment:
Required:
As per your view; User can choice here `cancel` as well. Let's take that in mind and write this method; what you say?
Take `cancle=False` default